cli/toolchain_cmd.go

```diff
@@ -86,15 +67,6 @@ func init() {
 		log.Fatal(err)
 	}
 
-	_, err = c.AddCommand("add",
```

Comment:
It is confusing to the 97% of users that aren't srclib toolchain developers. Several people have mentioned it. I think all it does is create a symlink. Are you OK with just manually creating the symlink as needed?

Other option is to rename it something like `link-dev-toolchain`, but IMO I prefer when simple operations like symlinks are exposed to the user instead of wrapped in an alias.